openedx/core/djangolib/nose.py

```diff
@@ -13,9 +13,6 @@ def setup_databases(self):
         """ Setup databases and then flush to remove data added by migrations. """
         return_value = super(NoseTestSuiteRunner, self).setup_databases()
 
-        # Delete all data added by data migrations. Unit tests should setup their own data using factories.
-        call_command('flush', verbosity=0, interactive=False, load_initial_data=False)
```

Comment:
I assume there was a reason this `flush` was added in the first place?
Did you check the history on this addition?